### ISSUE TITLE
Add Recommended Extensions md table generation

### ISSUE THREAD

**Issue by RandomFractals** [usernamehw/vscode-contributions#9]
I frequently add a section with recommended extensions that cover similar functionality to my extensions.

Those are typically saved in `./vscode/extensions.json` so that myself or other extension developers who want to send PR have a complete set of extensions installed and loaded while working in that tooling area.

For example: https://github.com/RandomFractals/geo-data-viewer/blob/master/.vscode/extensions.json

And that Recommended Extensions section might look like this:

https://github.com/RandomFractals/geo-data-viewer#recommended-extensions

What if as part of this extension, we could generate Recommended Extensions section too to help market other relevant extensions that add value to the tools we create.

Similarly, you could add an option to document ext. dependencies and just pull that info from ext. package.json.

**Comment by usernamehw** [comment]
I don't think `extensions.json` is for recommending similar extensions. It's a list of extensions to work (build, contribute) on current project, such as, `ESLint`, `editorconfig`,  `Chrome debugger`.

Example from official doc:

```js
{
  "recommendations": ["dbaeumer.vscode-eslint", "msjsdiag.debugger-for-chrome"]
}
```

https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions

I've never seen it being used like that, or the list of recommended extensions at the readme page, unless it's from author advertising their other extensions (related or unreleated).

**Comment by RandomFractals** [comment]
true. It's how I used it to install extensions I need to dev other extensions and compare similar tools. I'd say this is an extra bonus feature for those ext. devs who'd like to do the same.

And recommendations are not just for dev workspace setup. If you look at my geo data viewer list, it's ext's for data analytics and maps in vscode workspace setup, which is exactly what workspace ext. recommendations are for. Just depends on your flow.

P.S.: I usually remove those eslint recommendations. Most devs should already have those installed, and I find it annoying when vscode starts recommending I install linters I don't prefer and use when I load other extensions to sample.
